.github/workflows/build-linux.yml

```diff
@@ -14,98 +14,89 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        node-version: [ 16.x]
+        node-version: [16]
+        arch: [amd64, arm64, arm]
+        platform: [debian, alpine]
+        include:
+          # Set baselines per platform
+          - triple: "alpine-linux-musl"
+            platform: alpine
+            version: 3.16
+            libc: musl
+            gcc-install: clang lld
+            gcc: clang
+          - triple: "linux-gnu"
+            platform: debian
+            version: bullseye
```

Comment:
In DOckerfile it is Buster, here is bullseye. Is that intentional?

Reply:
Dockerfile should be Bullseye, as ubuntu 20.04 and Debian bullseye share the same glibc version (2.31). As the container runs the tests, it needs to have the same glibc version or a newer one.

I'll update the dockerfile